backend/src/controllers/ProjectController.ts

```diff
@@ -225,6 +227,27 @@ class ProjectController {
     return projects;
   }
 
+  public async findUserProjects(userId: string): Promise<ProjectDoc[]> {
+    let userMembers = await this.memberModel.aggregate([
```

Comment:
nit: off the top of my head, I think a simple `await this.memberModel.find({ user: userId })` will be faster than an `.aggregate()` call. (sort of like the tech model lookup in project search)

Reply:
Like, query from the perspective of the `Member` collection since the member is unique to the user and project